data/txHandler.go

```diff
@@ -179,7 +190,30 @@ func MakeTxHandler(opts TxHandlerOpts) *TxHandler {
 		)
 		handler.erl = rateLimiter
 	}
-	return handler
+
+	// prepare the transaction stream verifer
+	var err error
+	handler.streamVerifier, err = verify.MakeStreamVerifier(handler.streamVerifierChan,
+		handler.postVerificationQueue, handler.streamVerifierDropped, handler.ledger,
+		handler.txVerificationPool, handler.ledger.VerifiedTransactionCache())
+	if err != nil {
+		return nil, err
+	}
+	go handler.droppedTxnWatcher()
+	return handler, nil
+}
+
+func (handler *TxHandler) droppedTxnWatcher() {
+	for unverified := range handler.streamVerifierDropped {
```

Comment:
can't see where streamVerifierDropped is closed... so this goroutine never exits.

Reply:
Yes, this goroutine never exits. It starts with the opening of the channels, and the channels are never re-opened and never closed, so is the goroutine. 
You think this is a problem?